src/main/java/org/alfresco/repo/index/shard/ShardMethodEnum.java

```diff
@@ -77,4 +77,25 @@ public static ShardMethodEnum getShardMethod(String shardMethod)
                 return UNKOWN;
         }
     }
+    
+    /**
+     * Returns true if the method if any of the alias for EXPLICIT_ID methods, false otherwise
+     * @param shardMethod String representing the ShardMethod name
+     * @return true if the method is EXPLICIT_ID based
+     */
+    public static Boolean isExplicitIdMethod(String shardMethod)
```

Comment:
I would just return a primitive boolean. It's always going to be true or false I think.